app/models/casa_case.rb

```diff
@@ -48,6 +48,7 @@ class CasaCase < ApplicationRecord
   accepts_nested_attributes_for :casa_case_contact_types
   accepts_nested_attributes_for :court_dates
   accepts_nested_attributes_for :volunteers
+  accepts_nested_attributes_for :case_assignments, reject_if: proc { |attributes| attributes["volunteer_id"].blank? }
```

Comment:
This makes more sense to me compared to having the if statement at the controller level. 

I am not sure which is better 

```rb
accepts_nested_attributes_for :case_assignments, reject_if: proc { |attributes| attributes["volunteer_id"].blank? }
# or 
accepts_nested_attributes_for :case_assignments, reject_if: :all_blank
```